settings/deno.vim

```diff
@@ -13,7 +13,11 @@ augroup vim_lsp_settings_deno
       \     'implementations': v:true,
       \     'references': v:true,
       \     'referencesAllFunctions': v:true,
+      \     'test': v:true,
+      \     'testArgs': ['--allow-all'],
       \   },
+      \   'config': empty(lsp#utils#find_nearest_parent_file(lsp#utils#get_buffer_path(), 'tsconfig.json')) ? v:null : lsp#utils#find_nearest_parent_file(lsp#utils#get_buffer_path(), 'tsconfig.json'),
+      \   'internalDebug': lsp_settings#get('deno', 'internalDebug', v:false),
```

Comment:
[memo]
Add debug option, if set v:true, show deno lsp internal debug log to vim-lsp's log file.